docs/reference/monitoring/indices.asciidoc

```diff
@@ -2,8 +2,26 @@
 [[config-monitoring-indices]]
 == Configuring indices for monitoring
 
-<<indices-templates-v1,Index templates>> are used to configure the indices
-that store the monitoring data collected from a cluster.
+Sometimes the default index settings might not work for your situation. For
```

Comment:
@klacabane I'm trying to provide some context around why users might want to configure index settings. Is this too specific considering that the existing section about metricbeat focuses on setting number of shards/replicas and discourages users from changing other settings?

Reply:
I'd keep the context. ILM is a valid use case and while mappings removal/update can break dashboards, the wording here hints at _adding_ mappings which won't have side effects. We can maybe replace "other settings" with shards/replicas to align with metricbeat recommendation

Reply:
good idea!